src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -279,7 +280,8 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
                   classInfoUrl: portalData?.portalJWT?.class_info_url ?? "",
                   interactive: {
                     id: idNum,
-                    name: ""
+                    name: "",
+                    questionId
```

Comment:
To match my comments in the LARA PR, the id field here should be changed from `id: idNum` to just `id`. So it picks up the id prop.

I'd note that in the case of the AP these id values will be the old format `123-ManagedInteractive` instead of `managed_interactive_123`.  But as long as they are consistent in the AP answers the code shouldn't care.  It is unfortunate that we couldn't get them unified before the AP starting being used: https://docs.google.com/document/d/1vhRfcrxPRMtJRkHDqYbDPXKPDCqZV5JjjKoSmqB5zMg/edit# 
Implemented.